packages/portalnetwork/src/networks/network.ts

```diff
@@ -797,31 +788,37 @@ export abstract class BaseNetwork extends EventEmitter {
    * Do the random lookup on this node-id.
```

Comment:
Will you update these comments using the description from the PR to explain how the new refresh works?

Reply:
:heavy_check_mark: